.github/workflows/build_windows.yml

```diff
@@ -104,15 +91,11 @@ jobs:
     - name: Display version information
       run: ./ccextractorwinfull.exe --version
       working-directory: ./windows/Debug-Full
-    - name: Prepare artifacts
-      run: mkdir ./windows/artifacts
-    - name: Copy CCExtractor executable and pdb for artifact
-      run: cp ./windows/Debug-Full/ccextractorwinfull.exe ./windows/artifacts/; cp ./windows/Debug-Full/ccextractorwinfull.pdb ./windows/artifacts/
-    - name: Copy GUI for artifact
-      run: cp ./windows/Debug/ccextractorgui.exe ./windows/artifacts/; 
-    - name: Copy DLL's needed to artifact
-      run: cp ./windows/Debug-Full/*.dll ./windows/artifacts/
-    - uses: actions/upload-artifact@v1
+    - uses: actions/upload-artifact@v2
       with:
         name: CCExtractor Windows OCR and HardSubX Debug build
-        path: ./windows/artifacts
+        path: |
+          ./windows/Debug-Full/ccextractorwinfull.exe
+          ./windows/Debug-Full/ccextractorwinfull.pdb
+          ./windows/Debug/ccextractorgui.exe
```

Comment:
This triggers the Artifact to have 2 folders. If possible this should be avoided and all files placed in a single folder.